python/neuroglancer/chunks.py

```diff
@@ -44,6 +44,5 @@ def encode_jpeg(subvol):
     img.save(f, "JPEG")
     return f.getvalue()
 
-
 def encode_raw(subvol):
-    return subvol.tostring('C')
\ No newline at end of file
+    return subvol.tostring('F')
```

Comment:
This seems to work well for me. It allowed me to delete a few "magic" transposes that were in the code.
And have all the indexing in x,y,z,channels.